server/TelegramServerConfig.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+   Copyright (C) 2019 Alexey Minnekhanov <alexey.min@gmail.com>
```

Comment:
It would be more convenient to have a unified license header. Namely please add an empty line before and after the `This file is a part of...`